models/template_l1_FWA-1012VC.json

```diff
@@ -111,6 +118,7 @@
         "PciLong": "0000:00:15.0"
       },
       "logicallabel": "wwan0",
+      "cost": 0,
```

Comment:
We shouldn't default this to zero for wwan

Reply:
fixed. Made it 10